azurerm/helpers/validate/time.go

```diff
@@ -0,0 +1,46 @@
+package validate
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Azure/go-autorest/autorest/date"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+//todo, now in terraform helper, switch over once vended,
```

Comment:
Might be worth noting exactly what the helper is: https://godoc.org/github.com/hashicorp/terraform/helper/validation#ValidateRFC3339TimeString